events/class_tests/test_create_event_views.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+from events.class_tests.create_events import DATE_TIME_START, DATE_TIME_END, COLOR, TITLE
+from reminders.class_models.reminder_models import ReminderType
+from events.models import Event, Colors
+
+CREATE_EVENT_URL = "/event/create/"
+CREATE_EVENT_TEMPLATE = 'events/create_event.html'
+LOGIN_PAGE_TEMPLATE = 'login/register_login.html'
```

Comment:
Please add these variables to the conftest.

An example of this is
https://github.com/redhat-beyond/lets-meet/blob/830988ce721fc82eafb792293b51bb2145b6de79/reminders/class_tests/create_reminder_tests.py#L54-L57

and here is the definition:
https://github.com/redhat-beyond/lets-meet/blob/830988ce721fc82eafb792293b51bb2145b6de79/reminders/conftest.py#L36-L38

Reply:
Done